src/v/kafka/requests/create_topics_request.cc

```diff
@@ -29,6 +30,11 @@
 
 namespace kafka {
 
+bool is_not_supported(const ss::sstring& name) {
```

Comment:
how do we behave if a user tries to use a configuration option that isn't supported, but we haven't explicitly listed here (e.g. if kafka added it and we didn't update this list)? For that reason, it seems like a white list would be more appropriate than an exclusion list.

Reply:
good point @dotnwat.... @mmaslankaprv  moreover, our inclusion list will tend to be known and small. vs the exclusion list will tend to grow over time / unknown